catalog/ui/catalog-ui-search/src/main/webapp/properties.js

```diff
@@ -124,8 +124,16 @@ define(function (require) {
         },
         isDisableLocalCatalog: function(){
             return this.disableLocalCatalog;
+        },
+        isSearchHistoricalEnabled: function(){
```

Comment:
✏️  `isHistoricalSearchEnabled()` and `isArchiveSearchEnabled()`?

Reply:
fixed to match naming convention